static/js/people.js

```diff
@@ -73,7 +73,7 @@ exports.user_ids_string_to_emails_string = function (user_ids_string) {
 };
 
 exports.emails_strings_to_user_ids_string = function (emails_string) {
-    var emails = emails_string.split(',');
+    var emails = util.extract_pm_recipients(emails_string);
```

Comment:
Changed it to a more loose method, so that it wouldn't throw errors on strings like `a, b, c`, etc.